assets/base.css

```diff
@@ -1,151 +1,106 @@
-/* Derived from base colors (theme settings) */
+/* Color custom properties */
 
 :root,
 .color-background-1 {
   --color-foreground: var(--color-base-text);
-  --color-foreground-rgb: var(--color-base-text-rgb);
   --color-background: var(--color-base-background-1);
-  --color-background-rbg: var(--color-base-background-1-rgb);
-  --payment-terms-background-color: var(--color-base-background-1);
 }
 
 .color-background-2 {
   --color-background: var(--color-base-background-2);
-  --color-background-rbg: var(--color-base-background-2-rgb);
 }
 
 .color-inverse {
   --color-foreground: var(--color-base-background-1);
-  --color-foreground-rgb: var(--color-base-background-1-rgb);
   --color-background: var(--color-base-text);
-  --color-background-rbg: var(--color-base-text-rgb);
 }
 
 .color-accent-1 {
   --color-foreground: var(--color-base-solid-button-labels);
-  --color-foreground-rgb: var(--color-base-solid-button-labels-rgb);
   --color-background: var(--color-base-accent-1);
-  --color-background-rbg: var(--color-base-accent-1-rgb);
 }
 
 .color-accent-2 {
   --color-foreground: var(--color-base-solid-button-labels);
-  --color-foreground-rgb: var(--color-base-solid-button-labels-rgb);
   --color-background: var(--color-base-accent-2);
-  --color-background-rbg: var(--color-base-accent-2-rgb);
 }
 
 .color-foreground-outline-button {
   --color-foreground: var(--color-base-outline-button-labels);
-  --color-foreground-rbg: var(--color-base-outline-button-labels-rgb);
 }
 
 .color-foreground-accent-1 {
   --color-foreground: var(--color-base-accent-1);
-  --color-foreground-rbg: var(--color-base-accent-1-rgb);
 }
 
 .color-foreground-accent-2 {
   --color-foreground: var(--color-base-accent-2);
-  --color-foreground-rbg: var(--color-base-accent-2-rgb);
 }
 
-:root,
-.color-background-1,
-.color-background-2,
-.color-inverse,
-.color-accent-1,
-.color-accent-2 {
-  --color-foreground-75: rgba(var(--color-foreground-rgb), 0.75);
-  --color-foreground-70: rgba(var(--color-foreground-rgb), 0.7);
-  --color-foreground-55: rgba(var(--color-foreground-rgb), 0.55);
-  --color-foreground-50: rgba(var(--color-foreground-rgb), 0.5);
-  --color-foreground-40: rgba(var(--color-foreground-rgb), 0.4);
-  --color-foreground-30: rgba(var(--color-foreground-rgb), 0.3);
-  --color-foreground-20: rgba(var(--color-foreground-rgb), 0.2);
-  --color-foreground-10: rgba(var(--color-foreground-rgb), 0.1);
-  --color-foreground-8: rgba(var(--color-foreground-rgb), 0.08);
-  --color-foreground-4: rgba(var(--color-foreground-rgb), 0.04);
-  --color-foreground-3: rgba(var(--color-foreground-rgb), 0.03);
-}
```

Comment:
Removing those I guess makes it a little longer to update if ever needed when we go back and forth with designers.